src/send/SendConfirmationLegacy.tsx

```diff
@@ -475,17 +446,20 @@ function SendConfirmationLegacy(props: Props) {
   if (!account || !balance) {
     throw Error('Account is required')
   }
-  const feeProps: CalculateFeeProps = recipientAddress
-    ? {
-        feeType: FeeType.SEND,
-        account,
-        recipientAddress,
-        amount: amount.valueOf(),
-        currency,
-        balance: balance.toString(),
-        includeDekFee: !isDekRegistered,
-      }
-    : { feeType: FeeType.INVITE, account, amount, currency, balance: balance.toString() }
+
+  if (!recipientAddress) {
+    throw Error('Recipient address is required')
```

Comment:
this is one part that i was iffy about :/ there are 3 entry points to the SendConfirmationLegacy screen: 
1) scanning QR code if the tx is legacy - really unsure which real user flow this is?
2) IncomingPaymentRequestListItem - this seems to be the related to paying out a payment request, i think it's not possible for the recipient not to have an address if they've sent a payment request so i'm fairly sure this is safe
3) ValidateRecipientAccount - this seems to be part of the secure send flow, again i think it's not possible to reach this stage in the user journey and have no address on the recipient object.
would love any feedback on whether removing the InviteAndSendModal is a bad idea

Reply:
1. this is for secure send too, confirming the recipient by scanning a QR code
2. yes what's a bit confusing is payment requests can use the secure send flow. I don't recall the details about this. From reading the code it seems this is checked when the phone number is present in the payment request, to ensure it goes to the expected address, even though the requester address is there too.
3. yep this is what I understand too.

In brief, it seems safe to remove it 👍